docs/Agent/user-guide/man_page.md

```diff
@@ -450,44 +543,51 @@ Invoking `pbench-user-benchmark` with a workload generator as an argument will p
 - Gather the data from all the remote hosts and generate a `result.txt` file by running the tools' post-processing on the collected data.
 
-`<command-to-run>`
+`<command-to-run>`\
 A script, executable, or shell command to run while gathering tool data. Use `--`
 to stop processing of `pbench-user-benchmark` options if your command includes
 options, like
 
 > `pbench-user-benchmark --config string -- fio --bs 16k`
 
-**OPTIONS**  
-`-C`, `--config PATH`  
+**OPTIONS**
+
+[`-C`, `--config`] `<path>`\
 Path to the Pbench Agent configuration file.
 This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
 
-`--tool-group STR`  
+`--tool-group <tool-group>`\
 The tool group to use for data collection.
 
-`--iteration-list STR`  
-A file containing a list of iterations to run for the provided script;
-the file should contain one iteration per line. With a leading hash (`#`) character used for comments and blank lines are ignored.
-Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script.  
+`--iteration-list <file>`\
+A file containing a list of iterations to run for the provided script. The file
+must contain one iteration per line. Blank lines are ignored, and you can use a
+leading hash (`#`) character for comments. Each iteration line should use
+alpha-numeric characters before the first space to name the iteration, with the
+rest of the line provided as arguments to the script.
 _NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
-`--sysinfo STR[,STR...]`  
-Comma-separated values of system information to be collected; available: `default`, `none`, `all`, `ara`, `block`, `insights`, `kernel_config`, `libvirt`, `security_mitigations`, `sos`, `stockpile`, `topology`
+`--sysinfo STR[,STR]...`\
+Comma-separated values of system information to be collected; available:
+`default`, `none`, `all`, `ara`, `block`, `insights`, `kernel_config`,
+`libvirt`, `security_mitigations`, `sos`, `stockpile`, `topology`
 
-`--pbench-pre STR`  
-Path to the script which will be executed before tools are started.  
+`--pbench-pre <pre-script>`\
+Path to the script which will be executed before tools are started.
 _NOTE: --pbench-pre is not compatible with --use-tool-triggers_
 
-`--pbench-post STR`  
-Path to the script which will be executed after tools are stopped and postprocessing is complete.  
+`--pbench-post <post-script>`\
+Path to the script which will be executed after tools are stopped and
+postprocessing is complete.
 _NOTE: --pbench-post is not compatible with --use-tool-triggers_
 
-`--use-tool-triggers`  
-Use tool triggers instead of normal start/stop around script.  
-_NOTE: --use-tool-triggers is not compatible with --iteration-list,--pbench-pre, or --pbench-post_
+`--use-tool-triggers`\
+Use tool triggers instead of normal start/stop around script.
```

Comment:
"around script" could stand to be improved upon (or elided...).

At a minimum, we should replace "script" with `<command-to-run>`.

Beyond that, it would be better if the command description included a section on triggers as an alternative to the normal mechanism.  Then this text could simply reference that (or, at least, it would then make sense to replace "around script" with "mechanism" or "protocol" or similar).

Reply:
I took ten seconds to expand this, and realized we're not documenting the tool trigger *commands*. I'm not going to add them in this PR, but it's probably worth a Jira to remember that fun fact and fill in later. 😦 

[PBENCH-1179](https://issues.redhat.com/browse/PBENCH-1179)